src/_guides/libraries/objective-c-interop.md

```diff
@@ -84,6 +94,26 @@ all the functions, structs, enums, etc., that you're not interested in.
 For this example, we're only interested in `AVAudioPlayer`,
 so you can exclude everything else:
 
+```yaml
+  exclude-all-by-default: true
+  objc-interfaces:
+    include:
+      - 'AVAudioPlayer'
+```
+
+Since `AVAudioPlayer` is explicitly included like this,
+ffigen will exclude all other interfaces.
+The `exclude-all-by-default` flag tells ffigen to
+exclude everything else.
+The result is that nothing is included except `AVAudioPlayer`,
+and the things that it depends on, such as `NSObject` and `NSString`.
+So instead of several million lines of bindings,
+you end up with tens of thousands.
```

Comment:
Is this an exaggeration?

Reply:
Nope. Apple's internal libraries are huge.